core/templates/pages/blog-admin-page/services/blog-admin-data.service.ts

```diff
@@ -25,7 +25,8 @@ import { BlogAdminPageData, BlogAdminBackendApiService } from 'domain/blog-admin
   providedIn: 'root'
 })
 export class BlogAdminDataService {
-  dataPromise: Promise<BlogAdminPageData>;
+  // 'dataPromise' is null initially.
+  dataPromise: Promise<BlogAdminPageData> | null;
```

Comment:
Can we do non-null assertion here ? Removing null from the constructor. See https://github.com/oppia/oppia/pull/13459#discussion_r683037845